schemagen/schema.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 The Witness Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/in-toto/go-witness/attestation"
+	// this feels like an annoying way of importing them as it will need edited for each attestor added
```

Comment:
If you import `github.com/in-toto/go-witness/witness` will it pick up [imports.go](https://github.com/in-toto/go-witness/blob/main/imports.go)?

Reply:
yeah just done this all good 😄